packages/core-components/src/lib/unsorted/viz/Area.svelte

```diff
@@ -30,6 +30,10 @@
 
 	export let handleMissing = 'gap';
 
+	export let step = false; // shows step line. It can be true, false. Or 'start', 'middle', 'end' to configure turn point of step line.
```

Comment:
```suggestion
        /**
         * Configures the turn point of the step line.
         * @type {boolean | 'start' | 'middle' | 'end' }
         */
	export let step = false; // shows step line. It can be true, false. Or 'start', 'middle', 'end' to configure turn point of step line.
```